distribution/docs/receiver/application.properties

```diff
@@ -169,6 +169,11 @@ spring.artemis.password=
 camel.input.endpoint=activemq:openmrs.sync?connectionFactory=activeMqConnFactory&acknowledgementModeName=CLIENT_ACKNOWLEDGE&messageListenerContainerFactory=customMessageListenerContainerFactory
 # ----------------------------------------------------------------------------------------------------------------------
 
+# ****** Configuration of the ArtemisMQ endpoint to send payloads. {0} is the senderId, and will be set dynamically in runtime *****
+# Camel JMS Endpoint e.g. activemq:<queue-name>.{0}
+camel.output.endpoint=activemq:openmrs.sync.{0}
+# ----------------------------------------------------------------------------------------------------------------------------------
+
 
 # *********************** PGP configuration if encryption is necessary (OPTIONAL) **************************************
 #
```

Comment:
Please add the loggers for the new routes too and if you renamed any new routes update their loggers to match the new route names

Reply:
Ok, but no routes was created nor renamed in the receiver.